### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Sameony**
Created Landing page.
image

### CODE REVIEW

**Review by tanishq20 — APPROVED** [review]
Everything looks good. Good to merge.

File: Styles/main.css [hunk 1]
```diff
@@ -1 +1,29 @@
 @import url("https://mirinda-mewi.netlify.app/styles/main.css");
+@import url(/Styles/landing.css);
+* {
+  box-sizing: border-box;
+}
```
Comment (rahulcs754) [inline Styles/main.css:5]: you can use this in your component lib

**Review by rahulcs754 — APPROVED** [review]
all looks good. just small suggestion
I liked your design

File: Styles/main.css (hunk 1 above)
Comment (Sameony) [reply Styles/main.css:5]: using it now in component library gave some conflicts with the current design, so i will remodel it later. Thanks for the suggestion

**Pull request merged**